.travis.yml

```diff
@@ -3,13 +3,12 @@ sudo: false
 language: python
 
 cache:
-  apt: true
+  pip: true
   directories:
   - $HOME/.cache/pip
   - $HOME/.ccache
   - $HOME/.pip-cache
   - $HOME/third
-  - $HOME/nltk_data
```

Comment:
this is our largest cache directory, but i don't think it's very useful since we end up downloading with `force=True`